source/dem/pw_linear_force.cc

```diff
@@ -94,6 +95,13 @@ PWLinearForce<dim>::PWLinearForce(
       calculate_rolling_resistance_torque =
         &PWLinearForce<dim>::viscous_resistance;
     }
+
+  this->calculate_force_torque_on_boundary =
```

Comment:
Again shouldn't we call this function only when it is required? The same comment for the nonlinear force.

Reply:
These are member class and do not call any function. Do you mean that they should not be initialized if calculate_force_and_torque boolean is false?